test/functional/feature_governance.py

```diff
@@ -178,6 +178,13 @@ def run_test(self):
 
         assert_equal(payments_found, 2)
 
+        # v20 is expected to be activated at block 540
+        assert self.nodes[0].getblockcount() < 540
+        assert_equal(self.nodes[0].getblockchaininfo()["softforks"]["v20"]["active"], False)
+        assert_equal(self.nodes[0].getsuperblockbudget(530), satoshi_round("400.32798830"))
+        assert_equal(self.nodes[0].getsuperblockbudget(540), satoshi_round("4.00327990"))
```

Comment:
Note for reviewers: `develop` fails at this line (`getsuperblockbudget` still returns `400.32798830` even though v20 should be active at block 540 already)